src/main/java/model/Game.java

```diff
@@ -0,0 +1,93 @@
+package main.java.model;
+
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Scanner;
+
+import main.java.constant.GameConstants;
+
+public class Game {
+ private int playerCount = 0;
+ private int snakeCount = 0;
+ private int ladderCount = 0;
+ private Player[] players; 
+ private Map<Integer,Integer> snakes = new HashMap<>();
+ private Map<Integer,Integer> ladders = new HashMap<>();

```

Comment:
why do u need 2 maps ?

Reply:
yeah, true. now i see it that it wasn't required. I am doing same thing for both